views/default/plugins/event_calendar/settings.php

```diff
@@ -1,5 +1,17 @@
 <?php
 
+if (event_calendar_is_upgrade_available()) {
+	echo '<div class="elgg-admin-notices">';
+	echo '<p>';
+	echo elgg_view('output/url', array(
+		'text' => elgg_echo('event_calendar:upgrade'),
+		'href' => 'action/event_calendar/upgrade',
+		'is_action' => true,
+	));
+	echo '</p>';
+	echo '</div>';
+}
```

Comment:
I wonder if this is enough for admins to notice the need to upgrade. How about registering a handler for the ```'upgrade', 'system'``` hook and calling  ```elgg_add_admin_notice($id, $message)``` in it?